ocaml/idl/datamodel_pool.ml

```diff
@@ -8,6 +8,9 @@ open Datamodel_types
             "ha_disable", "Indicates this pool is in the process of disabling HA";
             "cluster_create", "Indicates this pool is in the process of creating a cluster";
             "designate_new_master", "Indicates this pool is in the process of changing master";
+            "set_repository", "Indicates this pool is in the process of setting repository";
+            "updates_sync", "Indicates this pool is in the process of syncing updates";
+            "updates_list", "Indicates this pool is in the process of listing updates";
```

Comment:
* `of setting up the repository` or `configuring the repository`
* `listening for updates`